Library/Homebrew/cmd/update-report.rb

```diff
@@ -133,7 +133,13 @@ def output_update_report
         "git", "-C", HOMEBREW_REPOSITORY, "tag", "--list", "--sort=-version:refname", "*.*"
       ).lines.first.chomp
 
-      new_repository_version = new_tag if new_tag != old_tag
+      if new_tag == old_tag
```

Comment:
```suggestion
      if old_tag.blank? || (new_tag == old_tag )
```

otherwise if `old_tag` is unset it'll use an empty string below.

Reply:
Ah, I missed that. That makes sense.